cpu/stm32w108/hal/micro/cortexm3/mfg-token.h

```diff
@@ -33,32 +36,28 @@
 /**
  * \brief Macro for translating token definitions into size variables.
  * This provides a convenience for abstracting the 'sizeof(type)' anywhere.
- *
- * \param name: The name of the token.
- *
- * \param type: The token type.  The types are found in token-stack.h.
  */
```

Comment:
I copied the docs from the first TOKEN_MFG instance now. Should I still remove TOKEN_MFG from doxygen?